mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/ModifiedPomExporter.java

```diff
@@ -0,0 +1,40 @@
+package org.arquillian.smart.testing.mvn.ext;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Writer;
+import org.arquillian.smart.testing.Logger;
+
+class ModifiedPomExporter {
+
+    static Logger logger = Logger.getLogger(ModifiedPomExporter.class);
+
+    private static String SMART_TESTING_POM_FILE = "smart-testing-pom.xml";
+
+    static void showPom(Model model) {
+        try (StringWriter pomOut = new StringWriter()) {
+            new MavenXpp3Writer().write(pomOut, model);
+            copyModifiedPomInTarget(pomOut, model.getProjectDirectory());
+        } catch (IOException e) {
+            throw new RuntimeException("Failed writing updated pom file: " + model.getPomFile().getAbsolutePath(), e);
+        }
+    }
+
+    static void copyModifiedPomInTarget(StringWriter modifiedPom, File project) throws IOException {
```

Comment:
This method also doesn't copy anything, it just prints the content of the modified pom to the file. It hasn't been stored anywhere, right?